nbs/examples/Getting_Started.ipynb

```diff
@@ -0,0 +1,435 @@
+{
```

Comment:
<p>Avoiding the Split train/test section would make the Getting Started easier. Optimal strategies for spliting (and cross validating) might be addressed in a different guide. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/323/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>